### ISSUE TITLE
undefined method `any?' for nil:NilClass

### ISSUE THREAD

**Issue by blackst0ne** [nepalez/rspec-sqlimit#10]
I have a simple test:

```ruby
it 'makes me happy' do
  expect { subject.call(foo: foo, bar: bar) }.not_to exceed_query_limit(1)
end
```

The `subject.call` does this:

```ruby
def call(foo:, bar:)
  foo.bars << bar
end
```

But rspec fails with

```bash
1) Example::ExampleQuery#call makes me happy
   Failure/Error: expect { subject.call(foo: foo, bar: bar) }.not_to exceed_query_limit(1)

   NoMethodError:
     undefined method `any?' for nil:NilClass
```

Instead the correct output should be 

```bash
  1) Example::ExampleQuery#call makes me happy
Failure/Error: expect { call_query }.not_to exceed_query_limit(1)

Expected to run maximum 1 queries
The following 3 queries were invoked:
  1) SAVEPOINT active_record_2 (0.251 ms)
  2) INSERT INTO "examples" ("foo_id", "bar_id", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id" (1.734 ms)
  3) RELEASE SAVEPOINT active_record_2 (0.769 ms)
```

This quick patch fixes the problem: 
```diff
diff --git a/home/blackst0ne/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/rspec-sqlimit-0.0.2/lib/rspec/sqlimit/reporter.rb b/tmp/reporter.rb
index a0141c075..884a7590a 100644
--- a/home/blackst0ne/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/rspec-sqlimit-0.0.2/lib/rspec/sqlimit/reporter.rb
+++ b/tmp/reporter.rb
@@ -28,7 +28,7 @@ module RSpec::SQLimit

     def line(query, index)
       prefix = (matcher && query[:sql] =~ matcher) ? "->" : "  "
-      binds = query[:binds].any? ? "; #{query[:binds]} " : ''
+      binds = query[:binds]&.any? ? "; #{query[:binds]} " : ''
       "#{prefix} #{index + 1}) #{query[:sql]}#{binds} (#{query[:duration].round(3)} ms)"
     end
   end
```

but I haven't dug why `query[:binds]` comes as nil to the `line` method, so I'm not sure this fix fixes the right place.

Could you take a look, please?

---

Ruby: 2.4.5
Rails: 4.2.11.1
rspec-sqlimit: 0.0.2

---

/cc @nepalez

**Comment by blackst0ne** [comment]
Any update here?

**Comment by nepalez** [comment]
Thank you @blackst0ne 
Fixed in v0.0.4

**Comment by blackst0ne** [comment]
@nepalez 

Thank you! :+1:
